R/JoinKeys.R

```diff
@@ -473,3 +479,19 @@ join_key <- function(dataset_1, dataset_2, keys) {
     class = "JoinKeySet"
   )
 }
+
+#' @rdname join_key
+#' @export
+#'
+#' @details
+#' `primary_key()` will create a primary key for a dataset. It is equivalent to
+#' `join_key(...)` and omitting `dataset_2` argument or giving it the same name
+#' `dataset_1`.
+#'
+primary_key <- function(dataset_1, keys) {
+  if (checkmate::test_character(keys) &&
+    !checkmate::test_names(names(keys), type = "unnamed")) {
+    stop("Primary keys parameter must be a unamed character vector: keys = c('A' = 'A') are not allowed")
+  }
```

Comment:
Should this accept `keys = c("A" = "A")` or should I just remove this check altogether?

Reply:
Maybe warning?

Reply:
Code was replaced